teuthology/openstack/__init__.py

```diff
@@ -883,12 +883,14 @@ def create_security_group(self):
         among instances created within the same tenant.
         """
         try:
-            self.run("security group show teuthology")
+            self.run("security group show teuthology && security group show teuthology-worker")
             return
         except subprocess.CalledProcessError:
             pass
         # FIXME: the following assumes "--name teuthology"
```

Comment:
I agree there is a better approach. I'd rather fix this together with what I mentioned in #64.
So yes, lets improve this code but maybe in a different PR. You ok with that @kshtsk ?